cli/fullservice.py

```diff
@@ -56,7 +58,7 @@ async def request(self, request_data: dict):
                     data=json.dumps(request_data),
                     headers={"Content-Type": "application/json"},
                 ) as resp:
-                    # print(resp.json)
+                    print_json(data=await resp.json(), indent=4)
```

Comment:
It seems like if this was a logging item before, I would think it should remain as a logging item?

So:
1. the output actually gets added into logs
2. esp if you want to redirect / save / quiet logs via `logging`

Or am I misunderstanding the python idioms here?

Seems [somewhat straightforward](https://rich.readthedocs.io/en/stable/console.html?highlight=print_json#printing-json) IIUC

Also, is there an advantage to moving the output here from lines 46-47?

Reply:
Cleaned this up a bit! Now it plays nicer with `logging` and prettyprints if the log level is set to `DEBUG`